docs/tutorial/03-fungible-tokens.mdx

```diff
@@ -512,22 +430,200 @@ Account storage is indexed with paths, which consist of a domain and identifier.
+        ?? panic("Could not borrow a reference to the owner's vault")
+
+    let temporaryVault <- vaultRef.withdraw(amount: 10.0)
+
+    // deposit your tokens to the Vault
+    vaultRef.deposit(from: <-temporaryVault)
+
+    log("Withdraw/Deposit succeeded!")
+  }
+}
+```
+
+<Callout type="info">
+Select account `0x01` as the only signer. <br/>
+Click the `Send` button to submit the transaction. <br/>
+This transaction withdraws tokens from the main vault and deposits them back to it.
+</Callout>
+
+This transaction is a basic example of a transfer within an account.
+It withdraws tokens from the main vault and deposits back to the main vault.
+It is simply to illustrate the basic functionality of how transfers work.
+
+You'll see in this transaction that
+you can borrow a reference directly from an object in storage.
 
 ```cadence
-self.account.save(<-create VaultMinter(), to: /storage/MainMinter)
+// Borrow a reference to the stored, private Vault resource
+let vaultRef = acct.borrow<&BasicToken.Vault>(from: /storage/CadenceFungibleTokenTutorialVault)
+    ?? panic("Could not borrow a reference to the owner's vault")
 ```
 
-You should also see that the `ExampleToken.Vault` and `ExampleToken.VaultMinter` resource objects are stored in the account storage.
-This will be shown in the Resources box at the bottom of the screen.
+This allows you to efficiently access objects in storage without having to load them,
+which is a much more costly interaction.
 
-You are now ready to run transactions that use the fungible tokens!
+In the real world, you'll likely be transferring tokens to other accounts.
+Capabilities allow us to accomplish this safely.
+
+## Ensuring Security in Public: Capability Security
+
+---
+
+Another important feature in Cadence is its utilization of **Capability Security.**
+This feature ensures that, while the withdraw function is public,
+no one except the intended user and those they approve of can withdraw tokens from their vault.
+
+Cadence's security model ensures that objects stored in an account's storage can only be accessed by the account that owns them.
+If a user wants to give another user access to their stored objects, they can link a public capability,
+which is like an "API" that allows others to call specified functions on their objects.
+
+An account only has access to the fields and methods of an object in a different account if they hold a capability to that object
+that explicitly allows them to access those fields and methods. 
+Only the owner of an object can create a capability for it.
+Therefore, when a user creates a Vault in their account, they only publish a capability
+that exposes the deposit function and the balance field.
+
+The withdraw function can remain hidden as a function that only the owner can call.
+
+As you can hopefully see, this removes the need to check `msg.sender` for access control purposes,
```

Comment:
```suggestion
This removes the need to check `msg.sender` for access control purposes,
```

Reply:
nit: This seems to be the first time that 'msg.sender' appears in this series of tutorial. Perhaps worth to mention this is the Ethereum idiom to handle access control

Reply:
Good point! I made that change